guides/source/action_view_overview.md

```diff
@@ -260,16 +251,18 @@ the `locals:` option, the template will raise an `ActionView::Template::Error`:
 <%= tag.div id: dom_id(product) do %>
   <h1><%= product.name %></h1>
 
-  <%# => raises ActionView::Template::Error %>
-  <% related_products.each do |related_product| %>
+  <%# => raises ActionView::Template::Error for `product_reviews` %>
+  <% product_reviews.each do |review| %>
     <%# ... %>
   <% end %>
 <% end %>
 ```
 
 ### Using `local_assigns`
 
-Each key in the `locals:` option is available as a partial-local variable through the [local_assigns][] helper method:
+There is a helper method [local_assigns][] available in each partial. You can access each key in the `locals:` option using this method. The value of `local_assigns[:some_key]` will be `nil` if the partial was not rendered with `:some_key` set.
```

Comment:
Suggestion to simplify: "Each partial has a helper method called local_assigns[]. You can use this method to access keys passed via the locals: option. If a key wasn't set when rendering the partial, local_assigns[:some_key] will return nil."

Reply:
reworded